src/override/malloc.cc

```diff
@@ -116,6 +116,36 @@ extern "C"
   }
 #endif
 
+#if !defined(__NetBSD__)
```

Comment:
So this isn't defined on NetBSD?

Reply:
ok so I got it backward, I thought, based on reallocarray above was not to export for systems having (e.g. freebsd has reallocarray).

Reply:
Yeah, I am not sure why `reallocarray` is not on `freebsd`,... in this file.  @davidchisnall is this intentional?